hardhat.config.js

```diff
@@ -47,6 +47,6 @@ module.exports = {
     timeout: 100000,
   },
   gasReporter: {
-    enabled: process.env.REPORT_GAS ? true : false,
+    enabled: !!process.env.REPORT_GAS,
```

Comment:
i prefer the older one so it's more explicit, seems less like a typo i feel

Reply:
Well `!!` is a standard way to do the type conversion in JS/TS, but they way it was makes it more readable